source/administration/upgrade.md

```diff
@@ -134,18 +134,20 @@ For any issues, Mattermost Enterprise Edition subscribers and trial license user
 
 ### Upgrade Enterprise Edition 
 
-#### Upgrade Enterprise Edition to 3.2.x
+#### Upgrade Enterprise Edition to 3.2.x and later
+
+Upgrading to `v3.2.x` and `v3.3.x` follows the procedure to [upgrade to `v3.1.x` and later](http://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition-to-3-1-x), but there is a breaking change where under `LdapSettings` in `config.json` `"FirstNameAttribute"` and `"LastNameAttribute"` are now required fields.
```

Comment:
Is `http://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition-to-3-1-x` missing the `s` from `https` ??